<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UtworzProjekt</class>
 <widget class="QDialog" name="UtworzProjekt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>312</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="nazwa_label">
      <property name="text">
       <string>Nazwa projektu:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nazwa_edit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="adres_label">
      <property name="text">
       <string>Nazwa bazy (*.sqlite)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="adres_edit"/>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="uzytkownik_przycisk">
        <property name="text">
         <string>Użytkownik</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="administrator_przycisk">
        <property name="text">
         <string>Administrator</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QListWidget" name="lista_wszyscy">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="lista_uzytkownicy">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lista_administratorzy">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QLabel" name="administratorzy_label">
        <property name="text">
         <string>Administratorzy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="zatwierdz_przycisk">
        <property name="text">
         <string>Zatwierdź</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="uzytkownicy_label">
        <property name="minimumSize">
         <size>
          <width>122</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Użytkownicy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="anuluj_przycisk">
        <property name="text">
         <string>Anuluj</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="blad_label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
